assets/js/modules/ads/index.js

```diff
@@ -75,6 +76,9 @@ export function registerModule( modules ) {
 				'google-site-kit'
 			),
 		],
+		SettingsDisconnectNoteComponent: isFeatureEnabled( 'adsPax' )
```

Comment:
Let's better stick with the `disconnectionNote` name as it better reflects the purpose of this property:
```suggestion
		disconnectionNote: isFeatureEnabled( 'adsPax' )
```